benchmarking/commons/baselines.py

```diff
@@ -51,6 +51,8 @@ class MethodArguments:
         arguments
     :param restrict_configurations: Only for simulator backend. If given, the
         scheduler is restricted to suggest configs from this list only
+    :param num_gpus_per_trial: Only for local backend and GPU training. Number
+        of GPUs assigned to a trial. Defaults to 1
```

Comment:
Would there be any downsides if we instead default to the maximum available?

Reply:
Yes! There is a trade-off between this and `n_workers`. If you use all GPUs for one trial, you can only have 1 worker.

In fact, I myself rather set this here to 1 and use the GPUs for different workers. This is the default.